### PR TITLE
Add Styling Section to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ronniegeraghty**
Added styling section to the CONTRIBUTING.md on 3rd Party Dependencies.

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -543,3 +543,18 @@ For information about breaking changes see [Breaking Change Rules](https://githu
...
+- **Commercial**: Commercial offerings that enable readers to learn from our content without unnecessary extra costs. Typically, the offering has some form of a community edition, or a free trial sufficient for its use in content. A commercial license may be a form of dual-license, or tiered license. Links to commercial components should be to the commercial site for the software, even if the source software is hosted publicly on GitHub (or similar).
```
Comment (jsquire) [inline CONTRIBUTING.md:556]:
```suggestion

- **Commercial**: Commercial offerings that enable readers to learn from our content without unnecessary extra costs. Typically, the offering has some form of a community edition, or a free trial sufficient for its use in content. A commercial license may be a form of dual-license, or tiered license. Links to commercial components should be to the commercial site for the software, even if the source software is hosted publicly on GitHub (or similar).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -543,3 +543,18 @@ For information about breaking changes see [Breaking Change Rules](https://githu
...
+- **Dual licensed**: Commercial offerings that enable readers to choose either license based on their needs. For example, if the offering has an OSS and commercial license, readers can  choose between them. [MySql](https://github.com/mysql/mysql-server) is an example of this license type.
```
Comment (jsquire) [inline CONTRIBUTING.md:557]:
```suggestion

- **Dual licensed**: Commercial offerings that enable readers to choose either license based on their needs. For example, if the offering has an OSS and commercial license, readers can  choose between them. [MySql](https://github.com/mysql/mysql-server) is an example of this license type.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -543,3 +543,18 @@ For information about breaking changes see [Breaking Change Rules](https://githu
...
+- **Tiered licensed**: Offerings that enable readers to use the license tier that corresponds to their characteristics. For example, tiers may be available for students, hobbyists, or companies with defined revenue  thresholds. For offerings with tiered licenses, strive to limit our use in tutorials to the features available in the lowest tier. This policy enables the widest audience for the article. [Docker](https://www.docker.com/), [IdentityServer](https://duendesoftware.com/products/identityserver), [ImageSharp](https://sixlabors.com/products/imagesharp/), and [Visual Studio](https://visualstudio.com) are examples of this license type.
```
Comment (jsquire) [inline CONTRIBUTING.md:558]:
```suggestion

- **Tiered licensed**: Offerings that enable readers to use the license tier that corresponds to their characteristics. For example, tiers may be available for students, hobbyists, or companies with defined revenue  thresholds. For offerings with tiered licenses, strive to limit our use in tutorials to the features available in the lowest tier. This policy enables the widest audience for the article. [Docker](https://www.docker.com/), [IdentityServer](https://duendesoftware.com/products/identityserver), [ImageSharp](https://sixlabors.com/products/imagesharp/), and [Visual Studio](https://visualstudio.com) are examples of this license type.
```

**Review by jsquire — APPROVED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -543,3 +543,21 @@ For information about breaking changes see [Breaking Change Rules](https://githu
 ## Debugging
 
 The libraries shipped out of this repo have [source link](https://docs.microsoft.com/dotnet/standard/library-guidance/sourcelink#using-source-link) enabled. Source link allows for symbols to be dynamically loaded while debugging, which allows you to step into the Azure SDK source code. This is often helpful when trying to step into Azure.Core code, as it is a package reference for most libraries. To enable using source link with the Azure SDK libraries in Visual Studio, you will need to check off Microsoft Symbol Servers as one of your Symbol file locations. Additionally, make sure that "Just My Code" is **_NOT_** enabled.
+
+## Styling
```
Comment (JoshLove-msft) [inline CONTRIBUTING.md:547]: Why is this called Styling?

Comment (ronniegeraghty) [reply CONTRIBUTING.md:547]:
```suggestion
## Samples
```

Comment (JoshLove-msft) [reply CONTRIBUTING.md]: Should we try to integrate this into the existing Samples section?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -543,3 +543,23 @@ For information about breaking changes see [Breaking Change Rules](https://githu
 ## Debugging
 
 The libraries shipped out of this repo have [source link](https://docs.microsoft.com/dotnet/standard/library-guidance/sourcelink#using-source-link) enabled. Source link allows for symbols to be dynamically loaded while debugging, which allows you to step into the Azure SDK source code. This is often helpful when trying to step into Azure.Core code, as it is a package reference for most libraries. To enable using source link with the Azure SDK libraries in Visual Studio, you will need to check off Microsoft Symbol Servers as one of your Symbol file locations. Additionally, make sure that "Just My Code" is **_NOT_** enabled.
+
+## Samples
+
+### Third-party dependencies
+
+Third party libraries should only be included in samples when necessary to demonstrate usage of an Azure SDK package; they should not be suggested or endorsed as alternatives to the Azure SDK
```
Comment (jsquire) [inline CONTRIBUTING.md:551]:
```suggestion
Third party libraries should only be included in samples when necessary to demonstrate usage of an Azure SDK package; they should not be suggested or endorsed as alternatives to the Azure SDK.
```

**Pull request merged**